class.jetpack-cli.php

```diff
@@ -63,7 +63,7 @@ public function status( $args, $assoc_args ) {
 
 			$cxntests->output_results_for_cli();
 
-			WP_CLI::error( __( 'Jetpack connection is broken.', 'jetpack' ) ); // Exit CLI.
+			WP_CLI::error( __( 'One or more tests did not pass. Please investigate!', 'jetpack' ) ); // Exit CLI.
```

Comment:
:) simple yet valid, this is no longer a connection indicator but could by a general sync issue.